src/components/notifications/Keep.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <div class="mb-3">
+        <label for="webhook-url" class="form-label">{{ $t("Host URL") }}</label>
```

Comment:
How Good is the UX to find the url/api-key?
=> Does this need a helptext how to find them?

Reply:
UX is good. Should be easy to navigate